version_test.go

```diff
@@ -1,14 +1,14 @@
-// +build unit
-
 package main
 
 import (
 	"testing"
 
 	"github.com/almighty/almighty-core/app/test"
+	skipper "github.com/almighty/almighty-core/test"
 )
 
 func TestAuthorizeLoginOK(t *testing.T) {
+	skipper.SkiptTestIfNotUnitTest(t)
```

Comment:
Well that's a good question. :)

I guess in the first iteration we can assume a DB is present if the ALMIGHTY_DB_HOST is set for instance..